code/__DEFINES/wiremod.dm

```diff
@@ -0,0 +1,85 @@
+#define PORT_INPUT_RECEIVE_DELAY 0.9 SECONDS
+
+/// Helper define that can only be used in /obj/item/circuit_component/input_received()
+#define COMPONENT_TRIGGERED_BY(trigger) (trigger.input_value && trigger == port)
```

Comment:
I don't like non-hygienic macros like this, `port` should be included in the macro.